requirements-full.txt

```diff
@@ -1,5 +1,8 @@
 tifffile>=2018.10.18
 scanimage-tiff-reader>=1.4.1.4
 neuroconv[video]>=0.4.6  # Uses the VideoCaptureContext class
+opencv-python-headless>=4.5.1.48
```

Comment:
Isn't this included in neuroconv video?

It might be a good idea to make it explicit though, do other extractors require it?

Reply:
I thought it would've been, but the tests were failing bc opencv wasn't installed.

Reply:
I would be concerned if this is related to the recent changes to pyproject.toml, when did the test start to fail?

Reply:
Ok, then it sounds unlikely. Still strange. Probably not worth finding out if this is working.